presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix accounting of raw input bytes in ScanFilterAndProjectOperator.
+* Fix an issue where server fails to start when two function namespace managers of the same type are specified.
+* Fix SQL function compilation error when input parameters contain lambda.
+* Fix ``IS DISTINCT FROM`` for long decimals.
+* Improve error handling for geometry deserialization edge cases.
+* Improve performance of array_join.
+* Improve efficiency of ``ST_Buffer``. The new implementation produces fewer buffer points on rounded corners, which will produce very similar but different results. It also better handles buffering with small (<1e-9) distances :func:`ST_Buffer`.
+* Improve OOM killer log output to put memory heavy nodes and queries first.
+* Improve efficiency by using JTS instead of Esri for many geometrical operations and fix incorrect calculation of extreme points in certain cases (:issue:`14031`).
+* Keep sum stat for string when min/max stat is too long in ORC writer.
+* Add support for get partition names by filter.
+* Add forms of :func:`approx_percentile` accepting an accuracy parameter.
+* Add new session property ``aggregation_partitioning_merging_strategy`` to control merging partitions when adding repartition remote exchange around aggregation node. Default option is ``LEGACY`` and can be overwritten to ``TOP_DOWN`` or ``BOTTOM_UP``.
+* Add ``IGNORE NULLS`` clause to various :doc:`/functions/window`.
```

Comment:
What does this look like in text? Please check the generated text and make sure they read fluently.

Reply:
its "Add IGNORE NULLS clause to various Window Functions."
the "Window Functions" goes to the windows.html page